doc/source/poppy_config.rst

```diff
@@ -0,0 +1,72 @@
+
+=========================
+Poppy Configuration Notes
+=========================
+
+Description
+-----------
+
+Please find below various poppy conf file configuration beside which in comments you will find the necessity of those configurations.
+These are some of the important configurations and also some regular configurations, not all the configuration information are explained below.
+
+Default configuration
+---------------------
+
+Make sure to keep the logging.conf in the correct directory
+
+[DEFAULT]
+verbose = True 			#To get verbosity for poppy server and worker
+debug = True 			#to get debug logs for poppy server and worker
+base_url = http://0.0.0.0:8888	#To have accessibility from anywhere
+
+log_config_append = /Users/<user>/.poppy/logging.conf   
+
+
+drivers configuration
+---------------------
+
+[drivers]
+providers = akamai 		#We currently have only Akamai as the provider
+
+
+storage configuration
+---------------------
+We currently use cassandra as our database.
+Set the migrations path correctly, to the location where you will find the migrations under the source code
+
+[drivers:storage:cassandra]
+migrations_path = /Users/<user>/poppy/poppy/storage/cassandra/migrations 	
+
+
+pecan configuration
+-------------------
+
+[drivers:transport:pecan]
+bind = 0.0.0.0			#Set bind address to 0.0.0.0
+port = 8888			#Set port to 8888 where requests will be served
+
+
+taskflow configuration
+----------------------
+
+We currently have our services running dockerized
+In your /etc/hosts file, make sure to set dockerhost to the ip of the dockerhost
+We currently have our services running dockerized
+
+[drivers:distributed_task:taskflow]
+jobboard_backend_host = "dockerhost"		#Set the host to dockerhost for this
+
+
+akamai provider configuration
+-----------------------------
+
+Set all the api keys, get these api keys from the password safe.
+Get the appropriate cert names and set them accordingly for testing, the ones listed above are the ones which are used for dev testing.
+
+[drivers:provider:akamai]
+
+contract_id : F-858L5F		#Set the contract id value from passwordsafe
```

Comment:
I think this is very useful -- but because` rackerlabs/poppy` is a public repo, we should prob omit this.  onewiki may have been a better choice for this after all.